fr.free.brouits.qspeakers.yml

```diff
@@ -0,0 +1,24 @@
+app-id: fr.free.brouits.qspeakers
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: qspeakers
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  - --device=dri
+  # Needs to save files locally
+  - --filesystem=xdg-documents
+rename-desktop-file: qspeakers.desktop
+rename-icon: qspeakers
+modules:
+  - name: qspeakers
+    buildsystem: qmake
+    sources:
+      - type: git
+        url: https://github.com/be1/qspeakers.git
+        branch: master
```

Comment:
```suggestion
```

the use of branch is not allowed. 1. with a commit it will break 2. without it break reproducibility